### PR TITLE
Pull request for pyez release 2.6.4

### PR DESCRIPTION
**Pull request by chidanandpujar**
What does this PR do?

Pull request for pyez release 2.6.4

What issues does this PR fix or reference?
Release 2.6.4
Enhancements done
Supported start_shell options to choose the shell types (sh or csh) #995
Bugs Fixed
Fixed Device facts current_re returns the SRX cluster  node0 and node1 details with cluster ID 16 #1135
Fixed upgrade ncclient version 0.6.13, updated requirements.txt to install ncclient==0.6.13 #1153
Fixed deprecation warning due to invalid escape sequences #1034
Fixed Unit tests test_sw_put_ftp failure #1165

Tests written?

Yes/No

### CODE REVIEW

File: INSTALL-RHEL.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Operating Systems
+---------------
+- Red hat Enterprise Linux 8.1 (Ootpa)
+
+#### Step 1: Install Python and PIP
+
+##### For Python 3.x:
+###### For RHEL:
+
+#### Step 1: Install Junos PyEZ
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:16]: It should be step 2, right ?

**Review by dineshbaburam91 — COMMENTED** [review]
Please address the comment

File: INSTALL-RHEL.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+These instructions were tested on a 64-bit systems from https://github.com/opscode/bento, and using the _Junos PyEZ_ library version 2.6.3.
+
+Operating Systems
+---------------
+- Red hat Enterprise Linux 8.1 (Ootpa)
+
+#### Step 1: Install Python and PIP
+
+##### For Python 3.x:
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:13]: Installation steps are missing

File: INSTALL-RHEL.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+###### For RHEL:
+
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:15]: Same here

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
+## Release 2.6.4 - 9 JUNE 2022
+### Features Added
+- Supported start_shell options to choose the shell types (sh or csh) #995
+
+### Bugs fixed:
+- Fixed Device facts current_re returns the SRX cluster  node0 and node1 details with cluster ID 16 #1135
+- Fixed upgrade ncclient version 0.6.13, updated requirements.txt to install ncclient==0.6.13 #1153
+- Fixed deprecation warning due to invalid escape sequences #1034
+- Fixed Unit tests test_sw_put_ftp failure #1165
+- Support for python 3.9
```
Comment (dineshbaburam91) [inline RELEASE-NOTES.md:10]: Please add it in Features added list

File: INSTALL-RHEL.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+### Installation on RHEL
+
+The following are instructions for setting up a system starting from a stock system images.
+
+These instructions were tested on a 64-bit systems from https://github.com/opscode/bento, and using the _Junos PyEZ_ library version 2.6.3.
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:5]: Is it 2.6.3 or 2.6.4 ?

File: INSTALL-RHEL.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Installing from GitHub
+
+Development code can be installed directly from GitHub based on any branch, commit, or tag.
+
+***Step 1 and 3 still required.***
+#### Alternate Step 4: Install Junos PyEZ from GitHub
+
+#### Step 4a: Install Git from OS packages
+    pip install git
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:36]: pip3

File: INSTALL-RHEL.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+#### Step 4b: Install Junos PyEZ from GitHub
+
+##### For Python 3.x:
+	    pip install git+https://github.com/Juniper/py-junos-eznc.git
```
Comment (dineshbaburam91) [inline INSTALL-RHEL.md:41]: same here

File: INSTALL-RHEL.md (hunk 1 above)
Comment (chidanandpujar) [reply INSTALL-RHEL.md]: thanks, updated 
#### Step 2: Install Junos PyEZ

File: INSTALL-RHEL.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+Enjoy!
+
+
+#### Installing from GitHub
+
+Development code can be installed directly from GitHub based on any branch, commit, or tag.
+
+***Steps 1 -2 are still required.***
+#### Alternate Step 4: Install Junos PyEZ from GitHub
```
[the remark this answers is not in the document]
Comment (chidanandpujar) [reply INSTALL-RHEL.md:33]: updated the step no's

File: INSTALL-RHEL.md (hunk 2 above)
Comment (chidanandpujar) [reply INSTALL-RHEL.md:13]: dnf install python3-pip

Comment (chidanandpujar) [reply INSTALL-RHEL.md:13]: dnf install python3-pip python3-wheel

File: RELEASE-NOTES.md (hunk 4 above)
Comment (chidanandpujar) [reply RELEASE-NOTES.md]: updated .
### Features Added
- Supported start_shell options to choose the shell types (sh or csh) #995
- Supported for python 3.9

File: INSTALL-RHEL.md (hunk 5 above)
Comment (chidanandpujar) [reply INSTALL-RHEL.md:5]: updated it to 2.6.4

File: INSTALL-RHEL.md (hunk 6 above)
Comment (chidanandpujar) [reply INSTALL-RHEL.md:36]: pip3 install git

File: INSTALL-RHEL.md (hunk 7 above)
Comment (chidanandpujar) [reply INSTALL-RHEL.md:41]: pip3 install git+https://github.com/Juniper/py-junos-eznc.git

**Review by dineshbaburam91 — APPROVED** [review]

**Pull request merged**